docs/src/markdown-pages/guides/permissions-configuration.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Paging @flowermm - I know you are in the middle of moving docs over and figured you'd like to check this one out.